docs/ignoring-files-folders-code.md

```diff
@@ -35,7 +35,32 @@ Without user customization, Semgrep refers to the following to define ignored fi
 In the absence of a user-generated `.semgrepignore`, Semgrep will refer to its repository's `.semgrepignore` default template:
 
 ```
-DEFAULT_SEMGREPIGNORE_TEXT
```

Comment:
Also not this one.

Reply:
huh, our build config updates these placeholders whenever you build the docs; that's pretty uncomfortable 😅

Reply:
Yeah the placeholder thing is pretty annoying. I think I have an idea for how to improve the situation.